docs/sources/operations/multi-tenancy.md

```diff
@@ -12,24 +12,39 @@ Tenant IDs can be any alphanumeric string that fits within the Go HTTP header
 limit (1MB). Operators are recommended to use a reasonable limit for uniquely
 identifying tenants; 20 bytes is usually enough.
 
-To run in multi-tenant mode, Loki should be started with `auth_enabled: true`.
+Loki defaults to running in multi-tenant mode.
+Multi-tenant mode is set in the configuration with `auth_enabled: true`.
 
-Loki can be run in "single-tenant" mode where the `X-Scope-OrgID` header is not
-required. In single-tenant mode, the tenant ID defaults to `fake`.
+When configured with `auth_enabled: false`, Loki uses a single tenant.
+The `X-Scope-OrgID` header is not required in Loki API requests.
+The single tenant ID will be the string `fake`.
 
 ## Multi-tenant Queries
 
-If run in multi-tenant mode, queries that gather results from multiple tenants can be enabled with
-the `multi_tenant_queries_enabled: true` configuration option in the querier. Once enabled, multiple
-tenant IDs can be defined in the HTTP header `X-Scope-OrgID` by concatenating them
-with `|`. For instance a query for tenant A and B can set `X-Scope-OrgID: A|B`.
+In multi-tenant mode, queries may gather results from multiple tenants.
+Set the querier configuration option `multi_tenant_queries_enabled: true` to enable queries to cross tenants.
+The query API request defines the tenants.
+Specify multiple tenants
+in the query request HTTP header `X-Scope-OrgID` by separating the tenant IDs with the pipe character (`|`).
+For example, a query for tenants `A` and `B` sets the header `X-Scope-OrgID: A|B`.
```

Comment:
I would say that a query for tenants `A` and `B` requires the header `X-Scope-OrgID: A|B`. Technically a query does not set any header.